test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/RequestTests.cs

```diff
@@ -27,6 +25,8 @@ namespace Microsoft.AspNetCore.Server.Kestrel.FunctionalTests
 {
     public class RequestTests
     {
+        const int semaphoreWaitTimeout = 2500;
```

Comment:
I'd also go with underscore. We aren't the most consistent with this unfortunately, but that ssems to be the most popular for `private const`.